src/main/java/de/qabel/desktop/storage/AbstractNavigation.java

```diff
@@ -248,8 +282,23 @@ private void autocommit() throws QblStorageException {
 		if (!autocommit) {
 			return;
 		}
+		if (autocommitDelay == 0) {
+			commit();	// TODO commitIfModified
+			return;
+		}
 
-		commit();
+		final long autocommitStart = System.currentTimeMillis();
+
+		scheduler.schedule(() -> {
+			try {
+				if (lastAutocommitStart != autocommitStart) {
+					return;
+				}
+				AbstractNavigation.this.commitIfChanged();
```

Comment:
... we could move it to an autocommitStrategy that is injected. Could getting rid of the AUTOCOMMIT_DELAY thing easier. But that won't change much of who decides when to commit.
Ideas?